### ISSUE TITLE
QGIS 3.10 "ERROR 6: EPSG PCS/GCS code [6-digit#] not found". Work-around

### ISSUE THREAD

**Issue by GitHubrist** [qgis/OLD-QGIS-Website-see-QGIS-Website-for-current-one#706]
TL;DR: Two files missing from /usr/share/gdal/2.2/ during upgrade to QGIS 3.10?: 
```
/usr/share/gdal/2.2/esri_extra.wkt
/usr/share/gdal/2.2/cubewerx_extra.wkt
```

Backstory:
I was having my typical difficulty working out how to set the projection coordinate systems for project and layers to get a layer from looking "squashed" to "normal."

However, an approach I was convinced had worked very recently failed to yield the desired results.

As a result of a sequence of lucky Events, I may have found a bug, at least in the upgrade process, at least in moving QGIS from 3.8 to 3.10 in Ubuntu 18.04.3. 
I started a new project, noting that project CRS: WGS84. 
I loaded a single vector layer, statesp020 from National Atlas 2M data, noting layer CRS: invalid projection.
Map looks squashed.
I changed project CRS to NAD83/Conus Albers and layer CRS to NAD83.
Although this has worked in the past, it had no effect this time.
**EVENT 1**: I had started QGIS from the command-line, which I rarely do, and noticed warnings:
```
ERROR 6: EPSG PCS/GCS code 102031 not found in EPSG support files.  Is this a valid EPSG coordinate system?
ERROR 6: EPSG PCS/GCS code 102003 not found in EPSG support files.  Is this a valid EPSG coordinate system?
ERROR 6: EPSG PCS/GCS code 37004 not found in EPSG support files.  Is this a valid EPSG coordinate system?
ERROR 6: EPSG PCS/GCS code 102017 not found in EPSG support files.  Is this a valid EPSG coordinate system?
ERROR 6: EPSG PCS/GCS code 102013 not found in EPSG support files.  Is this a valid EPSG coordinate system?
```
In researching "ERROR 6" I learned somewhere that codes below a certain number (32nnn?) originate from EPSG and larger numbers tend to be from ESRI.
I also found a reference to information about CRS codes being stored in a .csv file.
I ran a find on /usr: find /usr -type f -iname \*csv and found the likely directory to be: /usr/share/gdal/2.2/
I catted some .csv files there
**Event 2:** I idly catted epsg.wkt, at the same time not finding an esri.wkt file as I innocently expected. I was also surprised that .wkt is a text file; epsg.wkt contained just two lines:
```
include esri_extra.wkt
include cubewerx_extra.wkt
```
To my further surprise, neither of these files was co-present with epsg.wkt.
I did another find:
`find /usr -type f -iname esri_extra.wkt -o -iname cubewerx_extra.wkt`
**Event 3:** I found them quite accidentally in a package I'd only installed recently on a whim, brlcad:
```
/usr/brlcad/share/gdal/cubewerx_extra.wkt
/usr/brlcad/share/gdal/esri_extra.wkt
```
I made links to the two brlcad versions in the /usr/share/gdal/2.2/ directory, and restarted QGIS and after the same steps, my mapped looked "normal."

Although it's possible that the brlcad install moved the files from gdal to brlcad, I think it's more likely that they may have been inadvertently left out of, or removed from, the gdal/2.2 folder during the upgrade to QGIS 3.10.

**Comment by kannes** [comment]
Hey @GitHubrist  , thank you so much for reporting this issue. This repository however is for the QGIS *website*, not the software. Could you post the issue at https://github.com/qgis/QGIS/issues ? That would be great!

**Comment by DelazJ** [comment]
Thanks for spotting the report @kannes . As suggested, @GitHubrist please fill a report on the other repository. Closing.
